account_invoice_margin/demo/res_groups.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+
+    <record id="group_account_invoice_margin_security" model="res.groups">
+        <field name="users" eval="[(4, ref('base.user_admin'))]"/>
```

Comment:
Isn't better to put this as regular data and not demo one?

Reply:
done.